.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
+- docker
+sudo: required
 node_js:
 - '4.2'
 script:
 - npm test
+- make docker-build
```

Comment:
Your call. From our perspective, more tests are always better. I don't think a 40 - 45 minute wait on CI would bother us really (certainly not at this stage). All that said... that should probably be a separate PR?